src/controllers/comment.controller.js

```diff
@@ -13,14 +16,24 @@ class CommentController {
   static async addCommentOnTripRequest(req, res) {
```

Comment:
In service, I have a generic function which is `createComment`, can be called in any controller.
Because here it is comment controller that is why I called that function `addCommentOnTripRequest`.  When they want to add comments on a particular feedback, they can for example `addCommentAccommodation` function then call `createComment` from `comment.service`. 

That is why I make it like, did I do in a proper way?